src/components/LineChart/LineChart.jsx

```diff
@@ -496,14 +519,53 @@ const LineChart = createClass({
 		}, {});
 		const xPoints = _.chain(xPointMap).keys().map(_.toNumber).value();
 
+		if (_.isEmpty(data) || width < 1 || height < 1 || isLoading) {
+			const emptyMessageBodyProp = _.get(getFirst(this.props, LineChart.EmptyMessageBody), 'props');
+			const emptyMessageTitleProp = _.get(getFirst(this.props, LineChart.EmptyMessageTitle), 'props', {children: 'You have no data.'});
+
+			return (
+				<EmptyStateWrapper
+					isEmpty={_.isEmpty(data)}
+					isLoading={isLoading}
+				>
+					<EmptyStateWrapper.EmptyMessageBody {...emptyMessageBodyProp} />
+					<EmptyStateWrapper.EmptyMessageTitle {...emptyMessageTitleProp} />
```

Comment:
I think these can be simplified so that the EmptyMessage children don't even need to be passed in if they don't exist:

```
const emptyMessageBody = getFirst(this.props, LineChart.EmptyMessageBody);
```

```
{EmptyMessageBody || null}
```

Currently they are always passed it with or without extra props, but they don't have to be.